exo-test/bin/setup

```diff
@@ -1,2 +1,12 @@
 #!/usr/bin/env bash
-yarn install
+
+# check if yarn is installed
+command -V yarn >/dev/null 2>&1
```

Comment:
@alexdavid this is an old PR, Martin is no longer on the team. Can you please implement the changes yourself and 🚢 this?